stdlib/public/core/CMakeLists.txt

```diff
@@ -156,14 +156,9 @@ set(SWIFTLIB_SOURCES
   Zip.swift
   )
 set(GROUP_INFO_JSON_FILE ${CMAKE_CURRENT_SOURCE_DIR}/GroupInfo.json)
-set(swift_core_link_flags "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}")
-set(swift_core_framework_depends)
-set(swift_core_private_link_libraries)
 set(swift_stdlib_compile_flags "${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS}")
```

Comment:
I removed `swift_core_framework_depends` because it was only used in the `add_swift_library` call for `TARGET_SDKS ALL_APPLE_PLATFORMS`. Instead of setting it here, I just inlined it.

Similarly, `swift_core_link_flags` was just `SWIFT_RUNTIME_SWIFT_LINK_FLAGS`, except for Apple platforms, where `-all_load` was appended to it. So I just used `SWIFT_RUNTIME_SWIFT_LINK_FLAGS` for all calls to `add_swift_library`, except the call with `TARGET_SDKS ALL_APPLE_PLATFORMS` includes an `-all_load`.